mongodb/src/main/java/org/hibernate/ogm/datastore/mongodb/query/parsing/nativequery/impl/NativeQueryParser.java

```diff
@@ -105,6 +104,8 @@ public Rule Operation() {
 				Sequence( FindOne(), builder.setOperation( Operation.FINDONE ) ),
 				Sequence( FindAndModify(), builder.setOperation( Operation.FINDANDMODIFY ) ),
 				Sequence( Insert(), builder.setOperation( Operation.INSERT ) ),
+				Sequence( InsertOne(), builder.setOperation( Operation.INSERTONE ) ),
+				Sequence( InsertMany(), builder.setOperation( Operation.INSERTMANY ) ),
 				Sequence( Remove(), builder.setOperation( Operation.REMOVE ) ),
```

Comment:
@DavideD  please see to [MongoCollection](http://api.mongodb.com/java/current/com/mongodb/client/MongoCollection.html) .
Breafly, MongoCollection has not method "insert", but mongodb shell has. In this way, I must to **support all methods "insert" in the PR**.